x-pack/plugins/watcher/public/sections/watch_edit/components/watch_edit_execute_detail/watch_edit_execute_detail.html

```diff
@@ -95,24 +94,25 @@
           type="checkbox"
           ng-model="watchEditExecuteDetail.executeDetails.ignoreCondition"
           class="kuiCheckBox"
-        >
-        Ignore Condition (Condition is always true)
-      </label>
+          i18n-id="xpack.watcher.sections.watchEdit.executeDetail.conditionOverrides.ignoreConditionCheckboxLabel"
```

Comment:
the same as above => ignoreConditionLabel. Also please check other labels in this merge request